### PR TITLE
Search results: handle folders

### PR DESCRIPTION
**Pull request by nl0**
Screenshot 2020-12-17 at 18 23 28

### CODE REVIEW

File: catalog/app/components/SearchResults/SearchResults.js [hunk 1]
```diff
@@ -492,6 +500,23 @@ function ObjectHit({ showBucket, hit: { path, versions, bucket } }) {
...
+    path,
+    versions: [v],
+    bucket,
+  },
+}) {
+  const handle = { bucket, key: path }
+  return (
+    <Section>
+      <ObjectHeader {...{ handle, showBucket }} />
+      <Meta meta={v.meta} />
```
Comment (fiskus) [inline catalog/app/components/SearchResults/SearchResults.js:515]: I don't see meta for "dir" search query inside quilt-dev-null. How can I find directories with meta?

If there is no meta, ObjectHeader's margin-bottom appends to Section's padding-bottom, and space above and under dir's title are not equal.

File: catalog/app/components/SearchResults/SearchResults.js [hunk 2]
```diff
@@ -85,12 +88,12 @@ function HeaderIcon(props) {
   )
 }
 
-function ObjectHeader({ handle, showBucket, downloadable }) {
+function ObjectHeader({ handle, showBucket, downloadable = false }) {
   return (
     <Heading display="flex" alignItems="center" mb={1}>
```
Comment (fiskus) [inline catalog/app/components/SearchResults/SearchResults.js:93]: I think, you need `mb={1}` outside of `ObjectHeader`, and use `mb={0}` for directory without metadata.

**Review by fiskus — APPROVED** [review]

File: catalog/app/components/SearchResults/SearchResults.js (hunk 1 above)
Comment (nl0) [reply catalog/app/components/SearchResults/SearchResults.js:515]: > How can I find directories with meta?

i'm afraid the only option is creating them)

> If there is no meta, ObjectHeader's margin-bottom appends to Section's padding-bottom, and space above and under dir's title are not equal.

yeah i see, i'll fix that, thx

Comment (nl0) [reply catalog/app/components/SearchResults/SearchResults.js:515]: fixed

**Pull request merged**